packages/webapp/src/components/Finances/Transaction/ExpandedContent/LabourTable.jsx

```diff
@@ -0,0 +1,173 @@
+/*
+ *  Copyright 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+import React, { useState, useMemo } from 'react';
+import PropTypes from 'prop-types';
+import styles from './styles.module.scss';
+import { useTranslation } from 'react-i18next';
+import EnhancedTable from '../../../Table/v2';
+import StateTab from '../../../RouterTab/StateTab';
+import { useCurrencySymbol } from '../../../../containers/hooks/useCurrencySymbol';
+
+export default function LabourTable({ transaction }) {
+  const { t } = useTranslation();
+  const currencySymbol = useCurrencySymbol();
+
+  const [activeTab, setActiveTab] = useState('employee');
+
+  const employeeTotalHours = useMemo(
+    () => calculateTotalHours(transaction.items.EMPLOYEE),
+    [transaction],
+  );
+
+  const taskTotalHours = useMemo(
+    () => calculateTotalHours(transaction.items.TASK_TYPE),
+    [transaction],
+  );
+
+  const employeeColumns = useMemo(
+    () => [
+      {
+        id: 'employee',
+        label: t('SALE.LABOUR.TABLE.EMPLOYEE'),
+        Footer: <div className={styles.footer}>{t('SALE.LABOUR.TABLE.DAILY_TOTAL')}</div>,
+      },
+      {
+        id: 'time',
+        label: t('SALE.LABOUR.TABLE.TIME'),
+        format: (d) => `${d.time} h`,
+        align: 'right',
+        Footer: <b>{employeeTotalHours} h</b>,
+      },
+      {
+        id: 'labourCost',
+        label: t('SALE.LABOUR.TABLE.LABOUR_COST'),
+        format: (d) => {
+          return (
+            <span>
+              {currencySymbol}
+              {Math.abs(d.labourCost).toFixed(2)}
+            </span>
+          );
+        },
+        align: 'right',
+        Footer: (
+          <b>
+            {currencySymbol}
+            {Math.abs(transaction.amount).toFixed(2)}
+          </b>
+        ),
+      },
+    ],
+    [transaction, currencySymbol],
+  );
+
+  const tasksColumns = useMemo(
+    () => [
+      {
+        id: 'taskType',
+        label: t('SALE.LABOUR.TABLE.TASK'),
+        Footer: <div className={styles.footer}>{t('SALE.LABOUR.TABLE.DAILY_TOTAL')}</div>,
+        component: 'th',
+        columnProps: {
+          style: { width: '33%' },
```

Comment:
I'm sorry, but I didn't do a good job on writing stories, and the style is terrible... so it would be great if you could set whatever works best (if necessary)! If we need all columns to have the same width, 33% should be 33.33% or should be written using calc. (https://isotropic.co/how-to-1-3-of-100-33-333-in-css/)


https://github.com/LiteFarmOrg/LiteFarm/assets/33141219/4672eb31-3fd7-4181-b36d-1cbddcdbab07

To make the columns widths the same between the two tables, the same styles need to be added to `employeeColumns`, and `shouldFixTableLayout={true}` needs to be passed to the table component to make `width: 33%;` work. (it adds `table-layout: fixed;` to the table)
🙏🙏🙏

Reply:
Thank you!! I actually thought there was some reason I didn't understand for applying the styling to only the tasks table so I kept it!! 😂 

I noticed that adding the styling to both sets of columns is enough to fix the shifting, and on desktop views adding `shouldFixTableLayout` doesn't do anything additional. However, on mobile it does truncate the column values rather than letting the table be horizontally scrollable. Do you know if that's the preferred behaviour? Is that what you will use on your tables?

Reply:
Column widths might change depending on the length of the text that the columns hold on the desktop view...
The existing table which is being used in the labour page etc. truncates texts, so I tried not to change the behaviour when I created the new table. And yes, I added `shouldFixTableLayout` to my tables!